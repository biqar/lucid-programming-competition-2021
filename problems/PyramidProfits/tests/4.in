50
bd97b36e57 Jude 80
8844c15c4e 73bb6b28b4 70
c37495dc06 a7efedef3b 30
f3400c32e4 6a1c528626 15
1a3f3b353a 190bca8d62 70
3c14f971ee d376515445 25
8516c8f825 bd97b36e57 90
cc9d408e6e 773a82eb29 85
38d031d536 a7efedef3b 10
0fd66491df b7a9d41ccd 25
1694ad7b02 65901e885e 85
3a343b529a 771faab05a 55
de75ed29e5 1694ad7b02 95
fef8021866 1a3f3b353a 75
773a82eb29 63c910289e 10
4098c65988 a8ce7cfb0e 65
771faab05a bd97b36e57 65
e313048e43 1fb52560ad 85
a8ce7cfb0e 1fb52560ad 90
336ec49109 fef8021866 5
824efd8b5f Jude 50
6c68065739 b4f8a819ed 60
dd755b75bb 1a3f3b353a 75
fdb2dfd688 Jude 60
e4b592f3ff 63c910289e 20
ae26dc8d59 Jude 50
73bb6b28b4 Jude 40
899007abaa 63c910289e 10
7527eedcd1 336ec49109 30
63c910289e Jude 85
1b0b07f85c a87463f371 50
5e853ef0bc a7efedef3b 15
661f3b069a 1fb52560ad 30
d3bc2bad62 1fb52560ad 90
b7a9d41ccd d0cac620f4 60
6852aebfc5 dd755b75bb 65
d0cac620f4 Jude 60
a87463f371 dd755b75bb 25
a7efedef3b a6aaee2608 45
e71d1166fe 4098c65988 95
d606db32d1 1b0b07f85c 45
65901e885e dd755b75bb 10
6a1c528626 1fb52560ad 45
b4f8a819ed 8b1417fe7e 45
190bca8d62 1fb52560ad 40
8b1417fe7e 73bb6b28b4 90
3565c558bb 1fb52560ad 20
a6aaee2608 6a1c528626 75
1fb52560ad Jude 60
d376515445 63c910289e 80
100
a87463f371 697300
3c14f971ee 1516000
0fd66491df 1497200
3565c558bb 624600
b7a9d41ccd 1310700
f3400c32e4 297200
7527eedcd1 560400
dd755b75bb 434200
899007abaa 278700
6a1c528626 882300
e4b592f3ff 569200
dd755b75bb 227800
73bb6b28b4 776600
1694ad7b02 793000
e71d1166fe 220200
3c14f971ee 1000000
Jude 1535600
4098c65988 1429200
a6aaee2608 1334300
f3400c32e4 1145100
1fb52560ad 94500
1fb52560ad 1953500
6852aebfc5 724700
773a82eb29 78700
6c68065739 239700
d376515445 1779000
ae26dc8d59 1148500
0fd66491df 1996700
c37495dc06 703300
1b0b07f85c 324600
d606db32d1 241100
d3bc2bad62 1917400
661f3b069a 556100
e4b592f3ff 1116600
63c910289e 1177400
3565c558bb 1694600
ae26dc8d59 1684400
773a82eb29 242200
0fd66491df 1152100
3c14f971ee 1425400
5e853ef0bc 942500
bd97b36e57 1676000
8516c8f825 1471000
3a343b529a 1448100
1b0b07f85c 1351700
824efd8b5f 1203900
773a82eb29 1004300
336ec49109 1508600
fef8021866 1141700
336ec49109 755800
1b0b07f85c 397600
3565c558bb 455500
773a82eb29 287800
65901e885e 1337100
6c68065739 857400
5e853ef0bc 1489800
e71d1166fe 249700
bd97b36e57 706400
bd97b36e57 1186400
e313048e43 20700
dd755b75bb 1960700
6852aebfc5 1379500
cc9d408e6e 1109700
899007abaa 614100
ae26dc8d59 898000
a8ce7cfb0e 641800
f3400c32e4 122600
661f3b069a 1273500
1b0b07f85c 138600
7527eedcd1 770600
1a3f3b353a 1838300
899007abaa 1083300
1fb52560ad 1272600
65901e885e 593000
e4b592f3ff 582400
b7a9d41ccd 1200000
65901e885e 1877900
771faab05a 1546000
e4b592f3ff 1359800
d0cac620f4 1779300
1b0b07f85c 1024700
0fd66491df 1827100
6c68065739 531400
3565c558bb 19200
8844c15c4e 748800
e4b592f3ff 306600
fef8021866 881100
1694ad7b02 1571700
b7a9d41ccd 170400
8516c8f825 820900
Jude 468700
38d031d536 474100
190bca8d62 836700
1a3f3b353a 473200
a87463f371 862100
dd755b75bb 966100
7527eedcd1 207900
a87463f371 692900
fdb2dfd688 854700
3c14f971ee 1948000
