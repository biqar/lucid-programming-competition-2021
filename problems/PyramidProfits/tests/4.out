0fd66491df 582579
1694ad7b02 25326
190bca8d62 2053649
1a3f3b353a 1372073
1b0b07f85c 53074
1fb52560ad 16375737
336ec49109 21052
3565c558bb 335268
38d031d536 4320
3a343b529a 414157
3c14f971ee 1001198
4098c65988 509896
5e853ef0bc 33247
63c910289e 8574528
65901e885e 52450
661f3b069a 329328
6852aebfc5 172334
6a1c528626 931086
6c68065739 158290
73bb6b28b4 465636
7527eedcd1 2909
771faab05a 1142775
773a82eb29 151254
824efd8b5f 601950
8516c8f825 1650168
8844c15c4e 209664
899007abaa 167968
8b1417fe7e 322443
Jude 34026120
a6aaee2608 672226
a7efedef3b 272141
a87463f371 125731
a8ce7cfb0e 705502
ae26dc8d59 1865450
b4f8a819ed 105527
b7a9d41ccd 2712933
bd97b36e57 3876740
c37495dc06 19226
cc9d408e6e 80176
d0cac620f4 3264588
d376515445 4213314
d3bc2bad62 1035396
d606db32d1 1709
dd755b75bb 1786510
de75ed29e5 0
e313048e43 10557
e4b592f3ff 668882
e71d1166fe 156688
f3400c32e4 63378
fdb2dfd688 512820
fef8021866 710128
